### ISSUE TITLE
DSUmanager doesn't connect to the phone

### ISSUE THREAD

**Issue by JupiMay** [breeze2/dsu-manager-guides#1]
The app on the phone just says dsumanager connected but nothing happens

**Comment by breeze2** [comment]
Are the phone and PC on the same LAN?

**Comment by JupiMay** [comment]
Yes, and the connection with dolphin works. I have this problem only with dsumanager

**Comment by breeze2** [comment]
Run the command `ipconfig` on **cmd.exe** to get the ip of your PC.
Does it have the same prefix as the ip of your phone?

**Comment by JupiMay** [comment]
yes

**Comment by JupiMay** [comment]
could you help me?

**Comment by breeze2** [comment]
Currently I cannot reproduce it.
if it can connect to Dolphin, it should also can connect to DSUManager.
I don't know why it can't 😂.
Have you tried it with the full Wi-Fi signal ? Scan the QR code a few more times.

**Comment by JupiMay** [comment]
Still nothing...

**Comment by breeze2** [comment]
Does your PC have multiple IPs?
Can your show me your QR code?

**Comment by JupiMay** [comment]
Wdym with multiple ips?

**Comment by JupiMay** [comment]
![Screenshot 2023-05-30 184200](https://github.com/breeze2/dsu-manager-guides/assets/132830060/32b653ed-647e-4be9-b1c2-4a5f893fda28)

**Comment by lrutjens** [comment]
i have the same issue, i scan the code, the app says connected but nothing happens in the manager

**Comment by breeze2** [comment]
@lrutjens @JupiMay  Does it work?

**Comment by JupiMay** [comment]
yes, thank you so much!
